src/edit_view.rs

```diff
@@ -76,26 +69,115 @@ impl Font {
     }
 }
 
+#[derive(Clone)]
+pub struct ViewItem {
+    pub main_area: DrawingArea,
```

Comment:
Hmm..not quite sure if I'm so happy with `main_area` when this in in `EditView` and not in `MainWin`. Maybe `edit_area` would be more fitting?

Reply:
:+1: